lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -73,12 +76,36 @@ public function register_routes() {
 				'callback' => array( $this, 'update_item' ),
 				'permission_callback' => array( $this, 'update_item_permissions_check' ),
 				'args'     => array(
-					'post'         => array(),
-					'status'       => array(),
-					'content'      => array(),
-					'author'       => array(),
-					'author_email' => array(),
-					'author_url'   => array(),
+					'post'         => array(
```

Comment:
@rachelbaker we have an inconsistency here in the endpoints - some use the Schema to define all the `update_item` / `create_item` arguments, but here we are not doing that. One reason that may be is because we want to specify the `sanitize_callback` for each argument, which using the Schema does not allow us to do. Welcome to my recurring issue I've been wanting to solve for a while!